packages/teams-js/src/public/authentication.ts

```diff
@@ -202,6 +195,12 @@ export namespace authentication {
     });
   }
 
+  function throwIfStringNotValidHttpsUrl(link: string): void {
+    if (!isValidHttpsURL(new URL(link))) {
+      throw new Error(`${ErrorCode.INVALID_ARGUMENTS}: url must be https`);
```

Comment:
>url must be http [](http://example.com/codeflow?start=55&length=16)

I probably wouldn't say you _have_ to do this but since I already made one suggestion that I think you'll want to make, I will make this one too:
Since you do have a string here maybe it would be a little friendlier if you included the URL? Could help someone debug the problem if it ever happens. E.g.,

`${ErrorCode.INVALID_ARGUMENTS}: url must be https and given URL (${link}) is not`